app/views/layouts/application.html.erb

```diff
@@ -4,19 +4,23 @@
     <title>Dreamteam</title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="//cdn.muicss.com/mui-0.9.38/css/mui.min.css" rel="stylesheet" type="text/css"/>
```

Comment:
Why are we including `mui` from cdn if we already have the `mui-sass` gem?

Alternatively, you can also make use of https://rails-assets.org to include the library into your rails application, that way you can also remove the following line where you include the js part from the CDN as well.

see https://rails-assets.org/#/components/muicss--mui